hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/report/ReportManager.java

```diff
@@ -71,6 +76,11 @@ public void init() {
    */
   public void shutdown() {
     executorService.shutdown();
+    try {
+      executorService.awaitTermination(1, TimeUnit.DAYS);
+    } catch (Exception e) {
+      LOG.error("failed to shutdown Report Manager", e);
```

Comment:
The first letter of the first word as a capital letter maybe better.

Reply:
Do you want timeUnit to be Days?